myapp/loginJS.js

```diff
@@ -0,0 +1,92 @@
+
+// var usersJSON = '[{"userName":"combs","email":"kathycombs@quailcom.com","password":"orkman5951"},{"userName":"whitley","email":"kathywhitley@quailcom.com","password":"inson2079"},{"userName":"fletcher","email":"kathyfletcher@quailcom.com","password":"rederick8363"}]';
+
+// function color(){
+//   var x = document.getElementsByClassName( "input" );
+//   for( j in x ){
+//     x[j].style.backgroundColor="rgba( 51,153,255,0.2 )";
+//   }
+// }
+
+
+function changeInputColor(id) {
+  var inputElment = document.getElementById(id);
+  if( inputElment!== null){
+    inputElment.style.backgroundColor = "rgba( 51,153,255,0.2 )";
+    inputElment.style.border="1px solid rgb(51,153,255)";
+  }
+}
+
+function changeBackInputColor(id) {
+  var inputElment = document.getElementById(id);
+  if( inputElment !== null){
+    inputElment.style.backgroundColor = "#f2f2f2";
+    inputElment.style="border: 0px ";
+  }
+}
+
+
+
+function changeWrongInputColor(id){
+  var inputElment= document.getElementById(id);
+  if( inputElment !== null){
+    if(!inputElment.checkValidity()){
+    inputElment.style.backgroundColor = "rgba(255,0,0,0.2)";

```

Comment:
In all cases changeBackInputColor , changeWrongInputColor etc assing a class to the element instead of inline styles. Define some very well named classes and use them:

eg login-error, login-focus, login-success etc